includes/admin/reporting/reports.php

```diff
@@ -2313,7 +2313,6 @@ function edd_register_customer_report( $reports ) {
 					'lifetime_value_of_customer',
 					'average_customer_value',
 					'average_number_of_orders_per_customer',
-					'customer_average_age',
 					'most_valuable_customer',
```

Comment:
While we are here `most_valuable_customer` is not a valid tile and this can be removed as well.